src/components/card/card.pw.tsx

```diff
@@ -0,0 +1,353 @@
+
+        await expect(resultForColumnTwo).toEqual(countOfSecondColumn);
+      });
+    }
+  );
+
+  test("should check that the expected dataRole is set on the component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<CardComponent data-role="cypress" />);
+
+    const cardElement = await card(page);
+
+    await expect(cardElement).toHaveAttribute("data-role", "cypress");
+  });
+
+  test("should check interactive applies the expected styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<CardComponent interactive />);
+
+    const cardElement = await card(page);
+    await cardElement.hover();
+
+    await expect(cardElement).toHaveCSS("cursor", "pointer");
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.2) 0px 5px 5px 0px, rgba(0, 20, 30, 0.1) 0px 10px 10px 0px"
+    );
+  });
+
+  test("should allow custom boxShadow and hoverBoxShadow prop values and they apply the expected styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <CardComponent
+        boxShadow="boxShadow400"
+        hoverBoxShadow="boxShadow200"
+        interactive
+      />
+    );
+    const cardElement = await card(page);
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.04) 0px 10px 40px 0px, rgba(0, 20, 30, 0.1) 0px 50px 80px 0px"
+    );
+    await cardElement.hover();
+    await cardElement.waitFor();
+
+    await expect(cardElement).toHaveCSS("cursor", "pointer");
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.2) 0px 10px 20px 0px, rgba(0, 20, 30, 0.1) 0px 20px 40px 0px"
+    );
+  });
+
+  textAlignment.forEach((align) => {
+    test(`should check ${align} alignment applies the expected styling`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<CardTextAlignment align={align} />);
+
+      const columnCardElement = await columnCard(page);
+
+      await expect(columnCardElement).toHaveCSS("text-align", align);
+    });
+  });
+
+  ([375, 535, 777] as const).forEach((height) => {
+    test(`should check ${height} height is applied`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<CardComponent height={`${height}px`} />);
+
+      const cardElement = await card(page);
+
+      await assertCssValueIsApproximately(cardElement, "height", height);
+    });
+  });
+
+  test("should call onClick callback when a click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let setClickCounter = 0;
+    await mount(
+      <CardComponent
+        interactive
+        action={() => {
+          setClickCounter += 1;
+        }}
+      />
+    );
+
+    const cardElement = await card(page);
+
+    await cardElement.click({
+      force: true,
+    });
+
+    await cardElement.waitFor();
+    expect(setClickCounter).toEqual(1);
+  });
+});
+
+(["default", "large"] as CardProps["roundness"][]).forEach((roundness) => {
```

Comment:
Can this test be moved to above the action event one above please? Or even better, can the styling tests be placed in their own `describe` and the action event in its own? I'm of the mind now that they should be in meaningful `describes. Also, can the "S" in "Should" be changed to lower case to follow the same standard please.